apps/web/modules/availability/availability-view.tsx

```diff
@@ -203,18 +201,6 @@ export const AvailabilityCTA = () => {
     [searchParams]
   );
 
-  const { data } = trpc.viewer.organizations.listCurrent.useQuery();
-  const isOrgAdminOrOwner = (data && checkAdminOrOwner(data.user.role)) ?? false;
-  const isOrgAndPrivate = data?.isOrganization && data.isPrivate;
-
-  const canViewTeamAvailability = isOrgAdminOrOwner || !isOrgAndPrivate;
-
-  const toggleGroupOptions = [{ value: "mine", label: t("my_availability") }];
-
-  if (canViewTeamAvailability) {
-    toggleGroupOptions.push({ value: "team", label: t("team_availability") });
-  }
```

Comment:
we changed the design to remove this fetching + conditional toggle options because we are wasting a server fetch + Loader looks ugly since we don't know whether user will have 1 or 2 toggle options

Rather, we display two toggle options always, simple as that

AvailabilitySliderTable is already handling all cases (org, team, no team) beautifully thanks to @eunjae-lee

Reply:
<img width="386" alt="Screenshot 2025-04-23 at 9 45 06 AM" src="https://github.com/user-attachments/assets/06d5a096-edd9-4f64-9694-d1964822fee4" />

Final design

Reply:
appreciate your summary 🙏🏻